### PR TITLE
Fixes #261: Make websearch rss results similar to android and IOS

### LINKED ISSUE #261: Make websearh rss results UI similar to android iOS clients
**Actual Behaviour**

The current UI for websearch rss results uses a carousel like display

**Expected Behaviour**

UI must be made similar to android iOS clients which are also similar to the google assistant UI

**Steps to reproduce it**

Sample query : 
websearch : search for *
rss : stock quotes

**Screenshots of the issue**

screenshot from 2017-06-14 19 25 00

**Would you like to work on the issue?**

Anyone can take this up

 I tried a lot of modules including
[react-swipe](https://github.com/voronianski/react-swipe)
[nuka-carousel](https://github.com/FormidableLabs/nuka-carousel)
[react-slick](https://github.com/akiran/react-slick)
But faced issues with all of them, so I ended up using the current [react-responsive-carousel](https://github.com/leandrowd/react-responsive-carousel) module

### PR DESCRIPTION
**Pull request by rishiraj824**
Fixes issue #261 

Changes: 
- Removed Know more links
- Added responsive tiles

Demo Link: 
http://susi.surge.sh

Screenshots for the change: 
screenshot from 2017-06-21 05 20 32

### CODE REVIEW

**Comment by rishiraj824** [comment]
@uday96 Sure

File: src/components/ChatApp/MessageListItem.react.js [hunk 1]
```diff
@@ -78,7 +78,9 @@ function drawTiles(tilesData){
                 )}
               <div className='tile-text'>
                 <p className='tile-title'>
-                  <strong>{processText(tile.title)}</strong>
+                  <a href={tile.link} target='_blank'
+                  rel='noopener noreferrer'><strong>{processText(tile.title)}</strong>
+                  </a>
                 </p>
                 {processText(tile.description)}<br/>
                 <a href={tile.link} target='_blank'
```
Comment (uday96) [inline src/components/ChatApp/MessageListItem.react.js]: In google assistant, the entire tile is hyperlinked, not the heading. So we must hyperlink the entire <Paper> or <div> i guess

Comment (rishiraj824) [inline src/components/ChatApp/MessageListItem.react.js]: Okay will do @uday96

**Comment by rishiraj824** [comment]
@uday96 Done. Please check.

**Pull request merged**